src/foam/nanos/place/GooglePlaceService.js

```diff
@@ -0,0 +1,143 @@
+/**
+ * @license
+ * Copyright 2024 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.place',
+  name: 'GooglePlaceService',
+  documentation: `
+    Implement PlaceService using Google Place Service API.
+  `,
+
+  implements: [
+    'foam.nanos.place.PlaceService'
+  ],
+
+  javaImports: [
+    'foam.dao.DAO',
+    'foam.nanos.pm.PM',
+    'static foam.mlang.MLang.EQ',
+    'foam.nanos.logger.Loggers',
+    'foam.nanos.place.model.*',
+    'foam.nanos.place.configure.GooglePlaceServiceConfigure',
+    'org.apache.http.impl.client.HttpClients',
+    'org.apache.http.client.utils.URIBuilder',
+    'java.util.Arrays',
+    'java.util.ArrayList',
+    'foam.lib.json.JSONParser',
+    'org.apache.http.HttpEntity',
```

Comment:
@xuerongNanopay why the use of apache Http over native Java?

Reply:
Mainly because of org.apache.http.client.utils.URIBuilder. PlaceAPI build all datas/infos into the url query. URIBuilder is easier to do that